### DISCUSSION TITLE
A way to escape emoji?

### DISCUSSION THREAD

**Discussion by manoelpqueiroz** [Textualize/rich#3541] [Q&A] [answered]
Hi guys, I have a f-string in my application where I want to render some colour output with Rich, but also a unformatted string with an emoji shortcode:

```py
import textwrap
from rich import print

message = f'''
    Upload initial code to {scm_platform}:

        [bold red]$[/] [green]git[/] [yellow]add[/] [blue].[/]
        [bold red]$[/] [green]git[/] [yellow]commit[/] [blue]-m[/] [yellow]:tada: Initial commit"[/]
        [bold red]$[/] [green]git[/] [yellow]remote add[/] [red]origin[/] [blue]{scm_base_url}.git[/]
        [bold red]$[/] [green]git[/] [yellow]push[/] [cyan]-u[/] [blue]origin master[/]
'''
print(textwrap.dedent(message))
```

However, since I'm using Rich's `print` function, it will always render the `:tada:` emoji as :tada:. I have tried using `rich.markup.escape` as the [docs](https://rich.readthedocs.io/en/stable/markup.html#escaping) suggest, but to no avail.

Is it possible to escape the shortcode in this case?

**Comment by TomJGooding** [comment, marked as answer]
You can disable emojis using the [`Console`](https://rich.readthedocs.io/en/stable/console.html) class:

```python
from rich.console import Console

# Disable all emojis
console = Console(emoji=False)

# Disable emojis for specific prints
console.print("No emoji! :tada:", emoji=False)

```

**Reply by manoelpqueiroz** [reply]
Thanks! Didn't know it was that simple!
